vault/resource_ldap_secret_backend_static_role.go

```diff
@@ -64,9 +69,20 @@ var ldapSecretBackendStaticRoleFields = []string{
 	consts.FieldUsername,
 	consts.FieldDN,
 	consts.FieldRotationPeriod,
+	consts.FieldSkipImportRotation,
+}
+
+func updateLDAPStaticRoleResource(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	if _, ok := d.GetOk(consts.FieldSkipImportRotation); ok {
```

Comment:
We should use d.Get for booleans otherwise this can cause bugs. This is a limitation in the terraform SDKv2. See https://discuss.hashicorp.com/t/terraform-sdk-usage-which-out-of-get-getok-getokexists-with-boolean/41815